vmdk_plugin/E2E_Tests/e2e_verifyVolmProperties_test.go

```diff
@@ -0,0 +1,130 @@
+package e2e_test
+
+import (
+	"fmt"
+	TestUtil "github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/test_util"
+	"log"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+var esx string = ""
+var volSizes = []string{"200MB"}
+var vms = []string{os.Getenv("VM1"), os.Getenv("VM2")}
+var formatTypes = []string{"thin", "zeroedthick", "eagerzeroedthick"}
+
+func TestMain(m *testing.M) {
+	retCode := m.Run()
+	teardownFunction()
+	os.Exit(retCode)
+}
+
+func teardownFunction() {
+	fmt.Println("******clean-up******")
+	for i := 0; i < len(vms); i++ {
+		io := TestUtil.ExecuteCmd("docker ps", vms[i])
+		fmt.Println("in cleanup - ", io)
+		if strings.Contains(io, "busybox") {
+			TestUtil.ExecuteCmd("docker stop $(docker ps -a -q)", vms[i])
+			TestUtil.ExecuteCmd("docker rm $(docker ps -a -q)", vms[i])
+		}
+		cmd := "docker volume ls"
+		op := TestUtil.ExecuteCmd(cmd, vms[i])
+		if strings.Contains(op, "testVolume_0") {
+			cmd := "docker volume rm testVolume_0"
+			TestUtil.ExecuteCmd(cmd, esx)
+		}
+	}
+}
+
+func TestVolumeProperties(t *testing.T) {
+	defer teardownFunction()
+	esx = os.Getenv("ESX")
+	for j := 0; j < len(vms); j++ {
+		fmt.Println("running end_to_end tests...")
+		dkrVrsn := TestUtil.GetDockerVersion(vms[j])
+		fmt.Println("---- ", dkrVrsn)
+		if strings.Contains(dkrVrsn, "Docker version 1.11.") {
+			fmt.Println("Since docker version is less than 1.12, skipping test on this host")
```

Comment:
we have decided to skip the verification not the test, need to address that comment.